telosnetwork/open-block-explorer
# #494 | Number formatting

# Fixes #494 

## Description
This PR updates the way currency numbers are displayed sitewide according to the acceptance criteria. a new utility called `formatCurrency` has been added, and should be used in the future to display currency amounts. the PR also fixes a bug on the History tab where balances were being displayed like "5.000 TLOS TLOS". additionally it fixes several linting errors

## Test scenarios
- go to https://deploy-preview-652--obe-testnet.netlify.app/account/eztestnet123
- log in with cleos as `eztestnet123`
- click Staking (REX)
- go to History tab
    - values should be displayed properly (in prod they are displayed like "5.000 TLOS TLOS")
- generally click around the site
    - 0 values should be displayed without decimal places
    - TLOS values should be displayed with 4 precision
    - USD should be displayed with 2 precision
- go to https://deploy-preview-652--obe-testnet.netlify.app/account/faucet.tf
    - the large liquid value of the account should be displayed with a comma 

## Checklist:
